src/main/java/com/work/management/service/employee/impl/EmployeeServiceImpl.java

```diff
@@ -23,7 +23,7 @@
 import org.springframework.transaction.annotation.Transactional;
 
 @Service
-class EmployeeServiceImpl implements EmployeeService {
+public class EmployeeServiceImpl implements EmployeeService {
 
```

Comment:
No need for making it public.
Make the test case in test/java/com/work/management/service/employee/impl/EmployeeServiceImplTest.java